asdf_astropy/converters/transform/math_functions.py

```diff
@@ -0,0 +1,71 @@
+from .core import TransformConverterBase
+
+
+_MODEL_NAMES = [
+    "AbsoluteUfunc",
+    "AddUfunc",
+    "ArccosUfunc",
+    "ArccoshUfunc",
+    "ArcsinUfunc",
+    "ArcsinhUfunc",
+    "Arctan2Ufunc",
+    "ArctanUfunc",
+    "ArctanhUfunc",
+    "CbrtUfunc",
+    "CosUfunc",
+    "CoshUfunc",
+    "Deg2radUfunc",
+    # DivideUfunc is an alias for True_divideUfunc
+    "DivmodUfunc",
+    "Exp2Ufunc",
+    "ExpUfunc",
+    "Expm1Ufunc",
+    "FabsUfunc",
+    "Floor_divideUfunc",
+    "FmodUfunc",
+    "HypotUfunc",
+    "Log10Ufunc",
+    "Log1pUfunc",
+    "Log2Ufunc",
+    "LogUfunc",
+    "Logaddexp2Ufunc",
+    "LogaddexpUfunc",
+    # ModUfunc is an alias for RemainderUfunc
+    "MultiplyUfunc",
+    "NegativeUfunc",
+    "PositiveUfunc",
+    "PowerUfunc",
+    "Rad2degUfunc",
+    "ReciprocalUfunc",
+    "RemainderUfunc",
+    "RintUfunc",
+    "SinUfunc",
+    "SinhUfunc",
+    "SqrtUfunc",
+    "SquareUfunc",
+    "SubtractUfunc",
+    "TanUfunc",
+    "TanhUfunc",
+    "True_divideUfunc",
+]
```

Comment:
Is there a reason that we list all of these here explicitly instead of doing some automated pull of the names from `numpy` or `astropy.modeling.math_functions`?  Is there a test that makes sure this list stays up-to-date?  Or do we not serialize all the available math_functions?

Reply:
I listed these explicitly because I'm trying to avoid importing anything from astropy.modeling when this extension is loaded.  A worthy goal because that import is slow and the user will experience that as a lag in the asdf library, even when they're not using modeling.

Adding a test is a good idea though, I'll do that.

Reply:
That's a great reason.  Maybe just a test to make sure the coverage of `math_functions`, if it is intended to be complete, actually is.